src/jwe/index.ts

```diff
@@ -0,0 +1,76 @@
+import { JWEInvalid } from '../utils/errors';
+import type { JWEHeaderParameters, JWK } from '../types';
+import { IoReactNativeJwt } from '../utils/proxy';
+
+/**
+ * The EncryptJwe class is used to build and encrypt Flattened JWE objects.
+ *
+ * @example Usage
+ *
+ * ```js
+ * const jwe = await new EncryptJwe(
+ *   new TextEncoder().encode('It’s a dangerous business, Frodo, going out your door.'),
+ * )
+ *   .setProtectedHeader({ alg: 'RSA-OAEP-256', enc: 'A256GCM' })
+ *   .setAdditionalAuthenticatedData(encoder.encode('The Fellowship of the Ring'))
+ *   .encrypt(publicKey)
+ *
+ * console.log(jwe)
+ * ```
+ */
+export class EncryptJwe {
+  private _plaintext: String;
+
+  private _protectedHeader!: JWEHeaderParameters;
```

Comment:
*nitpick(style)* There's actually no need to prepend an undercore to mean the attribute is private.

Reply:
Without the underscore the attribute cannot be marked as private but will be public.
![Uploading Screenshot 2023-07-31 alle 09.16.39.png…]()

Reply:
It sounds very weird..